HISTORY.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 ---------
 
+2.3.0
+~~~~~
+
+* Add `--disable-mutation-types` and `--enable-mutation-types` to control what types of mutations are performed
+
```

Comment:
I am just guessing that this would probably be part of a V2.3.0. 
If it is uncommon that the changelog and version is maintained by the submitter of the PR I can revert this change.

Reply:
Any work I don't have to do it great :P Thanks!